scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
> IIRC that option was added to systemd/udev in a version later then is included in jessie

nah... doesn't seem like it, the version that gets installed is 215, and the kernel command line parameter is supposed to be present way before that...